src/middleware.ts

```diff
@@ -12,14 +12,20 @@ export function createMiddleware(
   return async (req, res, next) => {
     const serverOrigin = `${req.protocol}://${req.get('host')}`
 
+    // Ensure the request body input passed to the MockedRequest
+    // is always a string. Custom middleware like "express.json()"
+    // may coerce "req.body" to be an Object.
+    const requestBody =
+      typeof req.body === 'string' ? req.body : JSON.stringify(req.body)
+
     const mockedRequest = new MockedRequest(
       // Treat all relative URLs as the ones coming from the server.
       new URL(req.url, serverOrigin),
       {
         method: req.method,
         headers: new Headers(req.headers as HeadersInit),
         credentials: 'omit',
-        body: encodeBuffer(req.body),
+        body: encodeBuffer(requestBody),
```

Comment:
With msw@0.44, the input for the `body` is always an ArrayBuffer that encodes a plain string. Previously, it was possible passing a plain object as well, that's why the issue surfaced only now.